.github/actions/composite/setupGitForOSBotifyApp/action.yml

```diff
@@ -29,9 +29,9 @@ runs:
       shell: bash
       run: |
         if [[ -f .github/workflows/OSBotify-private-key.asc.gpg ]]; then
-          echo "::set-output name=key_exists::true"
+          echo "key_exists=true" >> "$GITHUB_OUTPUT"
```

Comment:
Why was this change necessary for this flow?

Reply:
`::set-output` is deprecated and will be deactivated soon